File: Comensa/Web/app/Refresh.php
<?php

class Refresh {
  public $contas;
  public $estabs;
  public $enderecos;
  public $produtos;
  public $promocoes;

  function setContas($contas) {
    $this->contas = $contas;
  }

  function setEstabs ($estabs) {
    $this->estabs = $estabs;
  }

  function setEnds($enderecos) {
    $this->enderecos = $enderecos;
  }

  function setProds($produtos) {
    $this->produtos = $produtos;
  }

  function setProms($promocoes) {
    $this->promocoes = $promocoes;
  }
}

?>

File: Comensa/Mobile/Tcc/app/src/main/java/com/example/tulio/tcc/Model/Refresh.java
package com.example.tulio.tcc.Model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by TULIO on 11-Jun-18.
 */

public class Refresh {
    @SerializedName("contas")
    private ArrayList<ContasModel> contas;
    @SerializedName("estabs")
    private ArrayList<EstabelecimentoModel> estabs;
    @SerializedName("enderecos")
    private ArrayList<EnderecoModel> ends;
    @SerializedName("produtos")
    private ArrayList<ProdutoModel> prods;
    @SerializedName("promocoes")
    private ArrayList<PromocaoModel> proms;

    public Refresh() {}

    public void inserirBd() {
        for(ContasModel c:contas)
            c.inserirBd();
        for(EstabelecimentoModel e:estabs)
            e.inserirBD();
        for(EnderecoModel e:ends)
            e.inserirBD();
        for(ProdutoModel p:prods)
            p.inserirBD();
        for(PromocaoModel p:proms)
            p.inserirBD();
    }
}
